packages/fxa-settings/src/models/hooks.ts

```diff
@@ -104,6 +104,15 @@ export function useIntegration() {
   }, [clientInfoState, productInfoState]);
 }
 
+export function useExperiments() {
+  const { experiments } = useContext(AppContext);
+  const { Features, Enrollments } = experiments;
```

Comment:
When I see capitalization in React, I think it's a type or it's a React component where I can use this like `<Features />`. Not sure what this is, but just pointing it out for when we look at getting experiment types done.

Reply:
The JSON properties inside Nimbus are capitalized which I found odd as well, but maybe in our mapping I'll just take that off. It caught me off-guard too. 😄